platform/core/poly-rdf/src/rdf.rs

```diff
@@ -46,53 +41,3 @@ pub fn rdf_update(query: SPARQLQuery, app_path: String) -> Result<(), RdfFailure
         Err(error) => Err(error),
     }
 }
-
-fn to_json_bytes(query_results: oxigraph::sparql::QueryResults) -> Result<Vec<u8>, RdfFailure> {
-    let mut results = Vec::new();
-    match query_results.write(&mut results, QueryResultsFormat::Json) {
-        Ok(_) => Ok(results),
-        Err(error) => Err(RdfFailure::map_evaluation_error(error)),
-    }
-}
-
-fn bytes_to_string(utf8_byte_array: Vec<u8>) -> Result<String, RdfFailure> {
-    match str::from_utf8(&utf8_byte_array) {
-        Ok(result) => Ok(result.to_string()),
-        Err(error) => Err(RdfFailure::map_utf8_error(error)),
-    }
-}
-
-#[cfg(test)]
```

Comment:
Are these test made elsewhere?

Reply:
No they don't really do anything as we are testing oxigraph more than anything else and they test themselves :D